src/Util/ExAuth.php

```diff
@@ -123,8 +159,6 @@ public function readStdin()
 	 */
 	private function isUser(array $aCommand)
 	{
-		$a = \get_app();
```

Comment:
Nice catch! I guess `\get_app()` is then deprecated?

Reply:
It definitly is.